pkg/apis/core/types.go

```diff
@@ -1639,6 +1639,13 @@ type VolumeMount struct {
 	// This field is beta in 1.10.
 	// +optional
 	MountPropagation *MountPropagationMode
+	// Expanded path within the volume from which the container's volume should be mounted.
+	// Behaves similarly to SubPath but environment variables from downward API are expanded prior to the mountpoint creation.
```

Comment:
are envvars from the downward API expanded, or envvars from the container spec?

Reply:
It makes use of the expansion routines which were part of the container and args env expansion. So it behaves the same way.

Reply:
then I think the docs from args are more correct: `Variable references $(VAR_NAME) are expanded using the container's environment.`